src/Console/Util/PhpProcess.php

```diff
@@ -63,7 +63,23 @@ public function start(callable $callback = null, array $env = null): void
         $phpConfig = new PhpConfig();
 
         $phpConfig->useOriginal();
+
+        // As of 1.3.2 xdebug-handler won't update $_ENV if it is in use.
+        // But Symfony's Process will happily import everything from $_ENV,
```

Comment:
Is it about these lines of code? 

https://github.com/symfony/process/blob/c43b190e78aa3cd0b17a9d12212b84590b4ead52/Process.php#L1644-L1648

Reply:
Exactly.